src/functionmap.jl

```diff
@@ -127,26 +128,125 @@ function At_mul_B!(y::AbstractVector, A::FunctionMap, x::AbstractVector)
         return y
     elseif ishermitian(A) # but !isreal(A)
         x = conj(x)
-        A_mul_B!(y, A, x)
+        mul!(y, A, x)
         conj!(y)
         return y
     else
         error("transpose not implemented for $A")
     end
 end
+Base.@propagate_inbounds function LinearAlgebra.mul!(y::AbstractVector, At::TransposeMap{<:Any,<:FunctionMap}, x::AbstractVector, α::Number, β::Number)
```

Comment:
This whole thing is currently needed because we can't use some "fallback", at least currently.